package.json

```diff
@@ -9,7 +9,7 @@
     "android": "expo start --android",
     "ios": "expo start --ios",
     "eject": "expo eject",
-    "test": "node ./node_modules/$npm_package_jest_preset/bin/jest.js --watchAll"
+    "test": "npx jest"
```

Comment:
I'll have a look.  But we should have at least 80% code coverage.  Green Up has an issue to write unit tests, so that is a known problem.   Having a watch on tests is important going forward so we'll automagically know when we break things.

Reply:
![image](https://user-images.githubusercontent.com/1043179/47834027-17338f80-dd74-11e8-89ce-4d0b2a6e4d87.png)

Reply:
Working for me.

Reply:
If you're having issues with modules not being found, try using yarn to install your packages.  npm has issues with expo and package linking doesn't always work.   running the `yarn` command often fixes these issues.